_includes/config-guide/custom-logger-handler-examples.md

```diff
@@ -0,0 +1,182 @@
+   <virtualType name="Magento\Payment\Model\Method\MyCustomLogger" type="Magento\Framework\Logger\Monolog">
+       <arguments>
+           <argument name="handlers" xsi:type="array">
+               <item name="debug" xsi:type="object">Magento\Payment\Model\Method\MyCustomDebug</item>
+           </argument>
+       </arguments>
+   </virtualType>
+   ```
+
+   where `Magento\Payment\Model\Method\MyCustomLogger` is the unique [virtual type]({{page.baseurl}}/extension-dev-guide/build/di-xml-file.html#virtual-types) identifier.
+
+1. Inject the declared virtual type (`Magento\Payment\Model\Method\MyCustomLogger`) into our object (`Magento\Payment\Model\Method\Logger`):
+
+   ```xml
+   <type name="Magento\Payment\Model\Method\Logger">
+       <arguments>
+           <argument name="logger" xsi:type="object">Magento\Payment\Model\Method\MyCustomLogger</argument>
+       </arguments>
+   </type>
+   ```
+
+1. The virtual class `Magento\Payment\Model\Method\MyCustomLogger` will be injected into the `debug` handler of the `$logger` property in the `Magento\Payment\Model\Method\Logger` class.
+ 
+   ```xml
+   ...
+   <argument name="handlers" xsi:type="array">
+       <item name="debug" xsi:type="object">Magento\Payment\Model\Method\MyCustomDebug</item>
+   </argument>
+   ...
+   ``` 
+
+   It will log exception messages into the `/var/log/payment.log` file.
+
+## Set up a custom file in the custom logger handler class
+
+1. Create a class that logs data. In this example, the class is defined in `app/code/Vendor/ModuleName/Logger/Handler/ErrorHandler.php`. 
+
+   ```php
+   <?php
+   /**
+    * @author Vendor
+    * @copyright Copyright (c) 2019 Vendor (https://www.vendor.com/)
+    */
+   namespace Vendor\ModuleName\Logger\Handler;
+
+   use Magento\Framework\Logger\Handler\Base as BaseHandler;
+   use Monolog\Logger as MonologLogger;
+
+   /**
+    * Class ErrorHandler
+    */
+   class ErrorHandler extends BaseHandler
+   {
+       /**
+        * Logging level
+        *
+        * @var int
+        */
+       protected $loggerType = MonologLogger::ERROR;
+
+       /**
+        * File name
+        *
+        * @var string
+        */
+       protected $fileName = '/var/log/my_custom_logger/error.log';
+   }
+   ```
+
+1. Define it for the specific handler type in the `di.xml`: 
+
+   For example, `Vendor\ModuleName\Logger\Handler\ErrorHandler` will be defined for the error handler:
+
+   ```xml
+   <virtualType name="MyCustomLogger" type="Magento\Framework\Logger\Monolog">
+       <arguments>
+           <argument name="handlers" xsi:type="array">
+               <item name="error" xsi:type="object">Vendor\ModuleName\Logger\Handler\ErrorHandler</item>
+           </argument>
+       </arguments>
+   </virtualType>
+   ```
+
+   where `MyCustomLogger` is the unique identifier of [virtual type]({{page.baseurl}}/extension-dev-guide/build/di-xml-file.html#virtual-types).
+
+1. Specify a class name that you will use in your custom logger handler.
```

Comment:
Specify the class name of the custom logger handler in the `type` definition. Use  the  virtual type name from the previous step as an argument for this type.

Delete the "For example" sentence.

Reply:
You do not understand a logic here. `Vendor\ModuleName\Observer\MyObserver` is a class where custom handler will be injected.